deploy/helm/charts/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for localpv.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+release:
+  version: "2.3.0"
+
+rbac:
+  # rbac.create: `true` if rbac resources should be created
+  create: true
+  # rbac.pspEnabled: `true` if PodSecurityPolicy resources should be created
+  pspEnabled: false
+
+# If false, openebs NDM sub-chart will not be installed
+openebsNDM:
+  enabled: true
+
+localpv:
+  name: localpv-provisioner
+  enabled: true
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: openebs/provisioner-localpv
+    tag: 2.3.0
+    pullPolicy: IfNotPresent
+  updateStrategy:
+      type: RollingUpdate
+  # If set to false, containers created by the localpv provisioner will run without extra privileges.
+  privileged: true
+  annotations: {}
+  podAnnotations: {}
+    ## Labels to be added to localpv provisioner deployment pods
+  podLabels:
+      name: openebs-localpv-provisioner
+  healthCheck:
+    initialDelaySeconds: 30
+    periodSeconds: 60
+  replicas: 1
+  enableLeaderElection: true
+  basePath: "/var/openebs/local"
+  resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  securityContext: {}
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+classes:
+  - name: localpv-device # Defines name of storage classe.
+    storageClass:
+      reclaimPolicy: Delete # Available reclaim policies: Delete/Retain, defaults: Delete.
+      isDefaultClass: true # set as default class
+      storageType: "device"
+      fsType: "ext4"
+      blockDeviceTag: ""
+
+  - name: localpv-hostpath # Defines name of storage classe.
+    storageClass:
+      reclaimPolicy: Delete # Available reclaim policies: Delete/Retain, defaults: Delete.
+      isDefaultClass: false
+      storageType: "hostpath"
+      # Path on the host where local volumes of this storage class are mounted under.
+      basePath: "/var/openebs/local"
```

Comment:
```suggestion
      basePath: "/var/openebs/local"
      # Custom node affinity label that will be used instead of hostnames
      nodeAffinityLabel: "openebs.io/node-affinity-value"
```

Reply:
I can just leave with place holder here with any value .. right ?

Reply:
yes, you can . Its label key that we are providing here.